renderer/containers/ModalStack.js

```diff
@@ -23,32 +23,68 @@ const Container = styled(animated.div)`
 const ModalContent = ({ type, closeModal }) => {
   const doCloseModal = () => closeModal()
   switch (type) {
-    case 'SETTINGS_FORM':
-      return <Settings onSubmit={doCloseModal} />
+    case 'SETTINGS':
+      return (
+        <Modal onClose={closeModal} p={0}>
```

Comment:
Not super clean that we have a lot of `<Modal onClose={closeModal}>` now, with only some of them having `p={0}`
Maybe it would be cleaner to have 
```
<Modal onClose={closeModal} {...getParams(modalype)}>
{getContent(modalype)}
</Modal>
```
but i don't have a strong opinion on this

Reply:
Agreed. I actually thought I changed this but seems the commit where I did got lost somewhere. I've updated this so that the Modal is fullscreen by default and it's up to users of the `Modal` component to add any padding that they want. So no more `padding={-4}` to remove default padding.